rse/grant-applicants.rst

```diff
@@ -0,0 +1,143 @@
+For grant applicants
+====================
+
+.. warning::
+
+   Grant applicants, please don't rely on this yet.  It will be
+   finalized soon.
```

Comment:
```suggestion
   Grant applicants, if you are planning to use Aalto Research Software Engineers service, feel free to contact us at rse-group at aalto.fi, or fill out our `request form <https://selfservice.esupport.aalto.fi/ssc/app#/order/2026/>`__.
```

Reply:
You are right this should be more positive, but also it needs a big disclaimer that it is still a draft and hasn't been reviewed by everyone yet.  My plan was to merge before review by others, since many of the people who need to review it can't be expected to look at PRs.  I'll merge and update.